spec/motion-spec/matchers/start_with_spec.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::StartWith" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.